crates/astria-composer/src/executor/mod.rs

```diff
@@ -386,9 +390,14 @@ impl Executor {
             Err(error) => error!(%error, "executor shutdown tasks failed to complete in time"),
         }
 
+        let number_of_submitted_bundles = if let Some(value) = bundles_drained {
+            value.to_string()
+        } else {
+            format!("more than {}", u64::MAX)
+        };
         if bundles_to_drain.is_empty() {
             info!(
-                number_of_submitted_bundles = bundles_drained,
+                %number_of_submitted_bundles,
```

Comment:
I think `Value for String`, so `%` can be omitted

Reply:
Strings are `Debug` formatted by default, so we need the `%` to force `Display`.